ha/cli/error.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+
+# This is temporary file,
+# To be removed once the "cortx cluster start" user story [EOS-16248] is started
+class Error(Exception):
```

Comment:
Inherit this from HAError and name it CliError.

Reply:
As mentioned in comment, this is temporary code and the entire file will be deleted once user story [EOS-16248]  is started; So I did not inherit this.

Reply:
Nyways will move the code to core.error and it can be deleted from there once EOS-16248 is done